src/components/PeopleFilters.tsx

```diff
@@ -22,8 +96,9 @@ export const PeopleFilters = () => {
             type="search"
             className="input"
             placeholder="Search"
+            value={searchQuery}
+            onChange={onSearchChange}
```

Comment:
The `onSearchChange` function should also update the search parameters to keep the URL in sync with the component state.